presto-docs/src/main/sphinx/functions/window.rst

```diff
@@ -121,3 +121,11 @@ Value Functions
     offset can be any scalar expression.  The default ``offset`` is ``1``. If the
     offset is null or larger than the window, the ``default_value`` is returned,
     or if it is not specified ``null`` is returned.
+
+Null Treatment
```

Comment:
Let's remove this section header and move the paragraph to be at the top of the "Value Functions" section. So it would look like
```
Value Functions
---------------

By default, null values are respected. If ...

.. function:: first_value(x) -> [same as input]